### PR TITLE
PT-1286 Integrate Tidepool master

### PR DESCRIPTION
**Pull request by franck-fourel**
- include metrics 
- K8s stuff

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -11,12 +11,28 @@ import queryString from 'query-string';
...
+import { Registry, Counter } from 'prom-client';
 import logMaker from './log';
 
 // const MemoryStore = require('memorystore')(session);
 
 const log = logMaker('app.js', { level: process.env.DEBUG_LEVEL || 'debug' });
 
+const { createTerminus } = require('@godaddy/terminus');
+
+const client = require('prom-client');
```
Comment (ydrisR) [inline app.js]: why do we mix import and require for prom-client?

**Review by ydrisR — COMMENTED** [review]
Looks good. Just one question :)

File: app.js (hunk 1 above)
Comment (franck-fourel) [reply app.js]: integrated as it is from tidepool.
Next commit may make it cleaner

**Review by ydrisR — APPROVED** [review]
LGTM

**Pull request merged**